waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/metrics/MonitoringConfigurationTestContext.java

```diff
@@ -27,4 +28,10 @@ GraphiteConfiguration graphiteConfiguration() {
     return conf;
   }
 
+  @Bean
+  PrometheusConfiguration prometheusConfiguration() {
```

Comment:
So, just to confirm, we now *always* expose a Prometheus endpoint? i.e. it's not something you can turn on or off?

Reply:
No I don't think so. The new Prometheus dependency will automatically create the PrometheusRegistry bean only. To expose the endpoint, when the jar is started up with some configuration, that configuration will need to include which endpoints to expose - this addition will happen here I think https://github.com/ExpediaGroup/apiary-federation/tree/master/templates

...subject to testing of course

Reply:
OK, make sense, we should document this then as per my other comment.